.github/workflows/ci.yml

```diff
@@ -12,8 +12,8 @@ jobs:
     strategy:
       matrix:
         php-version:
-          - "7.4"
           - "8.0"
+          - "8.1"
```

Comment:
Can you add the Symony versions to the build matrix as it is done [here](https://github.com/byWulf/apitk-common-bundle/pull/15/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03f)?

Reply:
I tried to but the result was a gamble. Composer always tried to checkout the depending apitk-bundles via SSH - which failed with a "could not authenticate against github.com" message. Even forcing composer to checkout via https via 

```
          git config --global url."https://github.com/".insteadOf 'git@github.com:'
```

didn't work and failed with the same errors.
However, when re-running the action, it sometimes worked. This seems like some caching issue or similar?
I invested about 3 hours into this and gave up on it for the time being.